github-actions/oss-readme-template/__tests__/readme.test.ts

```diff
@@ -70,6 +71,7 @@ For more info, visit our website at [https://gomomento.com](https://gomomento.co
         projectInfo: {type: ProjectType.OTHER},
         projectStatus: ProjectStatus.INCUBATING,
         projectStability: ProjectStability.EXPERIMENTAL,
+        usageExamplePath: './examples',
```

Comment:
This is very close, but I think we want the value here to be a path to a single file that has code we're going to slurp in.  So this might be something like `./examples/usage.ts`.